### ISSUE TITLE
Icon List - Text typography size not working when Theme Style link size is set

### ISSUE THREAD

**Issue by slamdive** [elementor/elementor#12021]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

The Icon List Widget - Text Typography size does not work if you have the Theme Style - Link Typography size set.
The Icon List Widget Text Typography size is not working independently from the Theme Style Link font size.

## Steps to reproduce

1) Using Elementor 'Theme Style' - set the Link typography size (16px for example).
2) Edit your page and add an Icon List widget.
3) Make sure to add a URL or # as the link in the Icon List widget (to enable the hyperlink <a>)
4) On the Icon List widget - go to Style / Text / Typography - and change the font size.
You will see nothing happens. 

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by dasignlady** [comment]
Same issue with Icon Box. The moment I enter a link, the title gets tiny and unresponsive to widget typography controls.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0

Feel free to update

Thanks!
